### DISCUSSION TITLE
Add race local time to the race details in live time

### DISCUSSION THREAD

**Discussion by MRAJEKO** [f1multiviewer/issue-tracker#162] [Feature requests]
Adding the time it on locally on track but delayed to the live timing next to the remaining session time.

**Comment by srshearer** [comment]
Moving comments from my dupe here…

> The addition of a clock to the Live Timing screen would help out quite a bit by giving context when there is an FIA message that includes what time something will happen. For example, "Session will resume at 15:35" means nothing when watching a session replay if we don't know the current time.

**Comment by LindaJuffermans** [comment]
I came here to request the same; with races across the pond (I'm in Europe) I will often watch qualifying the next morning.

**Comment by ianmdesign** [comment]
Quote from the suggestion I also left in discord:

**Simple suggestion im tossing in, track time in the live timing page shown to the second. Helps when f1tv has the time to session overlay during the preshow to help sync it.**
